ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Util.hs

```diff
@@ -450,3 +454,17 @@ withFuse (Fuse name m) (Electric io) = do
 newtype FuseBlownException = FuseBlownException Text
  deriving (Show)
  deriving anyclass (Exception)
+
+{-------------------------------------------------------------------------------
+  Type-safe boolean flags
+-------------------------------------------------------------------------------}
+
+-- | Type-safe boolean flags with type level tags
+--
+-- It is recommended to create pattern synonyms for the true and false values.
+-- Use 'coerce' to unwrap for use in e.g. if-statements.
```

Comment:
Couldn't we define a field name for the newtype and remove the need for using `coerce`?

Reply:
I just like the `coerce` use here, but I'm not opposed to an explicit field name either. I'll implement the explicit name approach.